### PR TITLE
Add Colin Breck's Influx days presentation

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: docs/src/main/paradox/other-docs.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Other documentation
 
+[From a time-series database to a key operational technology for the enterprise](https://youtu.be/3APiIht6oDY?t=1298)
```
Comment (ktoso) [inline docs/src/main/paradox/other-docs.md]: remove the ?t= though, people want to watch from the start ;)

**Comment by ktoso** [comment]
Fixed the link  https://github.com/akka/alpakka/pull/830/commits/872c13e34f59f000f466005ee30fa3eb4d339533

File: docs/src/main/paradox/other-docs.md (hunk 1 above)
Comment (ennru) [reply docs/src/main/paradox/other-docs.md]: Nooo, I put it there intentionally to point to the "Integration" part. It is easier to find the start than the Alpakka-explaining section.

Comment (ktoso) [reply docs/src/main/paradox/other-docs.md]: Oh, please mention such in PR next time :-) I'll add it back, sorry :)

**Comment by ktoso** [comment]
added it on master via github page editing... thanks

File: docs/src/main/paradox/other-docs.md (hunk 1 above)
Comment (ennru) [reply docs/src/main/paradox/other-docs.md]: Sorry, for not pointing it out.

**Pull request merged**